src/components/EntryForm.js

```diff
@@ -434,26 +433,29 @@ const EntryForm = ({ title, loo, center, children, ...props }) => {
 
           <Spacer mt={3} />
 
-          <ConditionalWrap
-            condition={!getValues('has-opening-times')}
-            wrap={(children) => <VisuallyHidden>{children}</VisuallyHidden>}
```

Comment:
visually hidden wrapper meant that it was still possible to tab to these elements.